infra/build/functions/request_build.py

```diff
@@ -56,45 +53,24 @@ def get_project_data(project_name):
   if not project:
     raise RuntimeError(
         f'Project {project_name} not available in cloud datastore')
-  project_yaml_contents = project.project_yaml_contents
-  dockerfile_lines = project.dockerfile_contents.split('\n')
-
-  return (project_yaml_contents, dockerfile_lines)
 
 
 def get_build_steps(project_name, image_project, base_images_project):
   """Retrieve build steps."""
-  project_yaml_contents, dockerfile_lines = get_project_data(project_name)
-  return build_project.get_build_steps(project_name, project_yaml_contents,
-                                       dockerfile_lines, image_project,
-                                       base_images_project)
+  # TODO(metzman): Figure out if we need this.
```

Comment:
I don't think this will work. These functions run in a context where the oss-fuzz repo is not available. We need to change that (as part of https://github.com/google/oss-fuzz/issues/5968#issuecomment-869441882), but probably not together with this current migration. 

Can you restore the previous logic to read project.yaml contents etc from Datastore and pipe that through the the right place (i.e. the `Project` class)?